### PR TITLE
2511 document featsuscription improving usability of subscription

### LINKED ISSUE #2511: Document: feat(suscription): Improving usability of subscription
This issue tracks the documentation update needed for the merged PR #18217.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/18217
Source PR Merged At: 2024-08-26T06:12:29Z

If it is a major improvement that deserves a new page or a new section in the documentation, please check if we should label it as an experiment feature.

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Description

+ Add command `show cursors` and `show subscription cursors`
+ Change the `op` column value from number to text
+ Change the default data scope of the result of fetch cursor

## Related code PR

https://github.com/risingwavelabs/risingwave/pull/18217

## Related doc issue

Resolve https://github.com/risingwavelabs/risingwave-docs/issues/2511

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/transform/subscription.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ DECLARE cursor_name SUBSCRIPTION CURSOR FOR subscription_name [since_clause];
 
 The `since_clause` is used to specify the starting point for reading data. By setting this clause, you can control the range of data that is returned, allowing you to retrieve only the incremental data or data starting from a specific time or event. 
 
-If you don’t specify the `since_clause`, the returned data will include both the historical data up to the time of declaration and the incremental data after declaration. If you want to specify it, here are the available choices:
+Below are the available choices for `since_clause`. If you don’t specify the `since_clause`, the returned data will just include the incremental data after declaration, which equals to the first choice below.
```
Comment (xxhZs) [inline docs/transform/subscription.md:73]: please add
DECLARE cursor_name SUBSCRIPTION CURSOR FULL， Will start consuming data from stock

Comment (ShanlanLi) [reply docs/transform/subscription.md:73]: Have added FULL, thanks.
image

Comment (WanYixian) [reply docs/transform/subscription.md:73]: @xxhZs Please take another look, will merge it if no questions 😄

File: docs/ingest/data-ingestion.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ The following statement will continuously import data from the Kafka topic and s
   FROM kafka_source;
   ```
 
-- Also, queries can be executed directly on the source, and **ad-hoc ingestion** will happen during the query's processing (more information in [directly query Kafka](/ingest/ingest-from-kafka.md#query-kafka-timestamp))
+- Also, queries can be executed directly on the source, and **ad-hoc ingestion** will happen during the query's processing (more information in [directly query Kafka](/ingest/ingest-from-kafka.md#query-kafka-timestamp)).
```
Comment (WanYixian) [inline docs/ingest/data-ingestion.md]:
```suggestion
- Also, queries can be executed directly on the source, and **ad-hoc ingestion** will happen during the query's processing, see more information in [directly query Kafka](/ingest/ingest-from-kafka.md#query-kafka-timestamp).
```

**Review by WanYixian — APPROVED** [review]
LGTM

**Pull request merged**